ncm-spma/src/main/perl/spma/yum.pm

```diff
@@ -363,11 +363,20 @@ sub expand_groups
 
     my $pkgs = Set::Scalar->new();
 
-    while (my ($group, $types) = each(%$groups)) {
+    while (my ($group_e, $types) = each(%$groups)) {
+        # A group can contain spaces and thus is escaped
+        my $group = unescape($group_e);
+        $self->debug(2, "Getting packages for group '$group'");
         my $what = join(",", grep($types->{$_}, keys(%$types)));
-        my $lst = $self->execute_yum_command([REPOGROUP, $what, $group],
-                                             "Group expansion", 1)
-            or return undef;
+        # repoquery command requires that each word in a group name be a separate parameter
```

Comment:
A mistake of mine, See https://github.com/quattor/configuration-modules-core/pull/813#issuecomment-232466766.